k8s/base/ipfs.yaml

```diff
@@ -0,0 +1,78 @@
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: ipfs
+  labels:
+    app: ipfs
+spec:
+  replicas: 1
+  serviceName: ipfs
+  selector:
+    matchLabels:
+      app: ipfs
+  template:
+    metadata:
+      labels:
+        app: ipfs
+    spec:
+      containers:
+        - name: ipfs
+          image: ipfs/ipfs
+          resources:
+            requests:
+              cpu: 250m
+              ephemeral-storage: 1Gi
+              memory: 512Mi
```

Comment:
I've seen kubo docs say as low as 2GB, but main docs say 6GB https://docs.ipfs.tech/install/command-line/#system-requirements. We should adjust the memory.